### ISSUE TITLE
Definer does not exist error

### ISSUE THREAD

**Issue by deepika05** [linuxserver/docker-mariadb#50]
```
root@mariadb:/# whoami                                                                                                        
root                                                                                                                          
root@mariadb:/# mysql -u root -p                                                                                              
Enter password:                                                                                                               
Welcome to the MariaDB monitor.  Commands end with ; or \g.                                                                   
Your MariaDB connection id is 9                                                                                               
Server version: 10.4.13-MariaDB-1:10.4.13+maria~bionic-log mariadb.org binary distribution                                    
                                                                                                                              
Copyright (c) 2000, 2018, Oracle, MariaDB Corporation Ab and others.                                                          
                                                                                                                              
Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.                                                
                                                                                                                              
MariaDB [(none)]> select user from mysql.user;                                                                                
ERROR 1449 (HY000): The user specified as a definer ('mariadb.sys'@'localhost') does not exist                                

```

------------------------------

## Expected Behavior
shown above is the first start. the command should list the users

## Current Behavior
see error above. this is a container on Synology Nas 218+ with only the MYSQL_ROOT_PASSWORD variable set when the container was created to eliminate any other conflicts/issues.

## Environment
**OS: Synology NAS 218+ Docker
**CPU architecture:** x86_64

**Comment by tobbenb** [comment]
I have no problems with that command here. This looks more like a support issue than a bug, so please use our forum or discord for support.

`root@a014c5f9b8a6:/# mysql -u root -p
Enter password: 
Welcome to the MariaDB monitor.  Commands end with ; or \g.
Your MariaDB connection id is 10
Server version: 10.4.13-MariaDB-1:10.4.13+maria~bionic-log mariadb.org binary distribution

Copyright (c) 2000, 2018, Oracle, MariaDB Corporation Ab and others.

Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.

MariaDB [(none)]> select user from mysql.user;
+--------+
| user   |
+--------+
| codimd |
| root   |
+--------+
2 rows in set (0.022 sec)

MariaDB [(none)]> 

`

**Comment by Pofilo** [comment]
Hello,
I had the same problem. 
I figure out the docker image of mariadb I was using (`mariadb:10.4.13`) did not have the same id than the last `mariadb:10.4.13` on docker hub.

With `docker image inspect <image_id>`:
Actual id: `"Id": "sha256:ac9c11a18222a7e80671fad178304eb6a50f958a4dbcacec9d1ec928ee2de8a0"`
Id I was using: `"Id": "sha256:c7bb8766aeddd79c67f567fe7a58af272cd388692d1a3a997f63809824c633c9"`

The dates were not the same too: `"Created": "2020-05-13T20:50:32.171154746Z"` VS `"Created": "2020-05-29T00:21:34.799784974Z"`.

I thought those tagged images were fixed, it's not the case ...

**TLDR: using the last version of `mariadb:10.4.13` fixed the problem.**
Thanks.
